src/Angor/Avalonia/Angor.Contexts.Funding/Founder/IFounderAppService.cs

```diff
@@ -16,5 +16,6 @@ Task<Result<TransactionDraft>> Spend(WalletId walletId, DomainFeerate fee, Proje
     Task<Result<IEnumerable<ReleaseableTransactionDto>>> GetReleasableTransactions(WalletId walletId, ProjectId projectId);
     Task<Result> ReleaseInvestorTransactions(WalletId walletId, ProjectId projectId, IEnumerable<string> investorAddresses);
     
+    Task<Result<ProjectSeedDto>> StartNewProjectAsync(WalletId walletId);
```

Comment:
We need to start also passing the network everywhere (not for this pr I guess) maybe we need a new type that will hold walletid, networkid, projectid